packages/@react-native-windows/cli/src/runWindows/runWindows.ts

```diff
@@ -15,16 +23,54 @@ import {Command, Config} from '@react-native-community/cli-types';
 import {runWindowsOptions, RunWindowsOptions} from './runWindowsOptions';
 
 import {autoLinkCommand} from './utils/autolink';
+import {totalmem, cpus} from 'os';
+import {execSync} from 'child_process';
+
+function getDiskFreeSpace(path: string | null) : Number {
+  const out = execSync(`dir /-C ${path}`).toString().split('\r\n');
+  const line = out[out.length - 2];
+  const result = line.match(/(\d+) [^\d]+(\d+) /);
+  if (result && result.length > 2) {
+    return Number(result[2]);
+  }
+  return -1;
+}
 
-function ExitProcessWithError(loggingWasEnabled: boolean): never {
+function SetExitProcessWithError(loggingWasEnabled: boolean): void {
   if (!loggingWasEnabled) {
     console.log(
       `Re-run the command with ${chalk.bold('--logging')} for more information`,
     );
   }
-  process.exit(1);
+  process.exitCode = 1;
 }
 
+function isMSFTInternal(): boolean {
+  return process.env.USERDNSDOMAIN !== undefined && process.env.USERDNSDOMAIN.endsWith('microsoft.com');
+}
+
+function getPkgVersion(pkgName: string) : string {
+  try {
+    const pkgJson = require(`${pkgName}/package.json`);
+    if (pkgJson.name === pkgName && pkgJson.version !== undefined) {
+      return pkgJson.version;
+    }
+  } catch { }
```

Comment:
Don't eat all errors. If you want to check if the require failed, you can check `ex.code === 'MODULE_NOT_FOUND`.

Reply:
same argument as above, since this is used for telemetry only, I don't want to ever fail for some any unforeseen reason